package.json

```diff
@@ -8,7 +8,7 @@
     "i18n"
   ],
   "devDependencies": {
-    "jasmine-node": "^1.14.5"
+    "jasmine-node": "^1.16.2"
```

Comment:
Is this necessary for this PR?
I am happy to update this on `master` with enough reason
But it doesn't seem to be related to the PR